File: AngularBankApiFrontEnd/src/app/model/address.ts
export class Address {
  id?: number;
  streetNumber?: string;
  streetName?: string ;
  city?: string;
  state?: string;
  zipCode?: string;

  constructor(id?: number, streetNumber?: string, streetName?: string, city?: string, state?: string, zipCode?: string) {
    this.id = id;
    this.streetNumber = streetNumber;
    this.streetName = streetName;
    this.city = city;
    this.state = state;
    this.zipCode = zipCode;
  }

  toString(): string {
    return `Address{id=${this.id}, streetNumber='${this.streetNumber}', streetName='${this.streetName}', city='${this.city}', state='${this.state}', zipCode='${this.zipCode}'}`;
  }
}

File: JpMorganBankApi/src/main/java/JpMorganApi/JpMorganBankApi/model/Address.java
package JpMorganApi.JpMorganBankApi.model;

import jakarta.persistence.*;



@Entity
public class Address {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "address_id")
    private Long id;



    private String streetNumber;



    private String streetName;



    private String city;



    private String state;



    private String zipCode;



    public Address(){

    }

//    public Address(Long id, String streetNumber, String streetName, String city, String state, String zipCode) {
//        this.id = id;
//        this.streetNumber = streetNumber;
//        this.streetName = streetName;
//        this.city = city;
//        this.state = state;
//        this.zipCode = zipCode;
//    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getStreetNumber() {
        return streetNumber;
    }

    public void setStreetNumber(String streetNumber) {
        this.streetNumber = streetNumber;
    }

    public String getStreetName() {
        return streetName;
    }

    public void setStreetName(String streetName) {
        this.streetName = streetName;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getZipCode() {
        return zipCode;
    }

    public void setZipCode(String zipCode) {
        this.zipCode = zipCode;
    }

    @Override
    public String toString() {
        return "Address{" +
                "id=" + id +
                ", streetNumber='" + streetNumber + '\'' +
                ", streetName='" + streetName + '\'' +
                ", city='" + city + '\'' +
                ", state='" + state + '\'' +
                ", zipCode='" + zipCode + '\'' +
                '}';
    }
}
